packages/tupaia-web/src/layout/TopBar/TopBar.tsx

```diff
@@ -33,8 +32,7 @@ const Header = styled.header<{
   padding: 0 0.625em;
 
   > * {
-    background-color: ${({ $primaryColor, theme }) =>
-      $primaryColor || theme.palette.background.default};
+    background-color: ${({ theme }) => theme.projectCard.background};
```

Comment:
Same as above - need to keep structured as before, but you can update theme variables